doc/joystick_control_teleoperation/joystick_control_teleoperation_tutorial.rst

```diff
@@ -12,6 +12,8 @@ If you haven't already done so, make sure you've completed the steps in `Getting
 
 Running the Code
 ----------------
+This jogging should only be used for simulated robots because some IK solvers can cause a joint jump. To jog a real robot, use the Jacobian-based `Real-Time Arm Jogging <../arm_jogging/arm_jogging_tutorial.html>`_ instead.
```

Comment:
this approach doesn't use IK actually - it uses RRT Connect etc to plan for the goal set by the joystick. I don't think this sentence is fair for this approach, there's just different beasts